charts/sam/values.yaml

```diff
@@ -0,0 +1,9 @@
+image:
+  repository: gcr.io/broad-dsp-gcr-public/sam
+  # Override the image tag to deploy by setting this variable.
+  # If no value is set, the chart's appVersion will be used.
+  # tag: latest
+gateway:
```

Comment:
worth a comment on what this controls, or is the style to go and lookup where it's used?

Reply:
Sure